### PR TITLE
Added Ripple Type Button

### PR DESCRIPTION
**Pull request by HADES-01**
@shahednasser  I have added Ripple type button as mentioned in [BUTTON_IDEAS.md](https://github.com/shahednasser/sbuttons/blob/master/BUTTON_IDEAS.md)

Issue: #1

### CODE REVIEW

File: BUTTON_IDEAS.md [hunk 1]
```diff
@@ -15,35 +15,37 @@ When adding ideas, please add them in the right category (or add a new one if yo
...
-|status  | Button name    | Description      | Submitted by |
-|--------|----------------|------------------|--------------|
-| &#9745;| ~~**Plus button**~~| A button just with a plus icon | | 
-| &#9745;| ~~**Buy me a coffee button**~~|  | |
-| &#9744;| **Icon Button**| A button with icons enlargement and shrink effect on hover | [Souvik Guria](https://github.com/devSouvik)
+| status  | Button name                    | Description                                                | Submitted by                                 |
+| ------- | ------------------------------ | ---------------------------------------------------------- | -------------------------------------------- |
+| &#9745; | ~~**Plus button**~~            | A button just with a plus icon                             |                                              |
+| &#9745; | ~~**Buy me a coffee button**~~ |                                                            |                                              |
+| &#9744; | **Icon Button**                | A button with icons enlargement and shrink effect on hover | [Souvik Guria](https://github.com/devSouvik) |
```
Comment (shahednasser) [inline BUTTON_IDEAS.md:32]: Change "&#9744;" to "&#9745;"

**Review by shahednasser — CHANGES_REQUESTED** [review]
Can you run "npm run compile" again? It seems there's something wrong with the compilation.

File: BUTTON_IDEAS.md (hunk 1 above)
Comment (shahednasser) [reply BUTTON_IDEAS.md:32]: It should be "&#9745" instead of "&#9744"

Comment (HADES-01) [reply BUTTON_IDEAS.md:33]: @shahednasser I think it's fine now.

**Review by shahednasser — CHANGES_REQUESTED** [review]
There's a weird problem were in "_ripple.less" the class's name is `ripple-btn` however in the compiled sbuttons.css it's `riple-btn` with one p. Are you sure you're compiling it correctly?

File: assets/js/buttons-examples.js [hunk 2]
```diff
@@ -60,6 +60,11 @@ $(document).ready(function () {
           label: "Invert",
           classes: "invert-btn",
         },
+        {
```
Comment (shahednasser) [inline assets/js/buttons-examples.js:63]: Can you place the button in the correct alphabetical order under "Animated"?

File: src/components/animated/_ripple.less [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  &.pink-btn {
+    .rippleBtn(@pink);
+  }
+
+  &.green-btn {
+    .rippleBtn(@green);
+  }
+
+  &.yellow-btn {
+    .rippleBtn(@yellow);
```
Comment (shahednasser) [inline src/components/animated/_ripple.less:47]: Can you add for yellow something similar to in "fill-btn" where before hover the color of text is yellow, but then on hover it becomes black? You can use the example from there.

**Review by shahednasser — CHANGES_REQUESTED** [review]

**Comment by HADES-01** [comment]
@shahednasser I have made the required changes.

**Review by shahednasser — CHANGES_REQUESTED** [review]
It looks great but the color before hover is still not applied. There's this piece of code in `src/components/_basic.less`:

```
.yellow-btn {
  .colorRule(@yellow);

  &:not(.fill-color-btn) {
    color: @darkText !important;
  }
}
```

can you add to the `&:not(.fill-color-btn)` part `, &:not(.ripple-button)`?

**Comment by HADES-01** [comment]
@shahednasser I have made the required changes.

**Comment by HADES-01** [comment]
@shahednasser I have added what you said and fixed the merge conflicts.

File: src/components/_basic.less [hunk 4]
```diff
@@ -63,7 +63,8 @@ a.sbtn {
 .yellow-btn {
   .colorRule(@yellow);
 
-  &:not(.fill-color-btn) {
+  &:not(.fill-color-btn),
```
Comment (shahednasser) [inline src/components/_basic.less:66]: After testing it, I think this should be &:not(.fill-color-btn):not(.ripple-btn). Sorry for the mistake!

**Review by shahednasser — CHANGES_REQUESTED** [review]

File: src/components/_basic.less (hunk 4 above)
Comment (HADES-01) [reply src/components/_basic.less]: @shahednasser Did the changes, please review them.

**Comment by shahednasser** [comment]
Great! Thank you!

**Pull request merged**